node/src/components/chainspec_loader.rs

```diff
@@ -150,6 +150,13 @@ impl Display for NextUpgrade {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct CurrentRunInfo {
```

Comment:
Can you add a comment to this?

Reply:
Not sure about the name. Maybe `MostRecentUpgrade`? :confused:

Reply:
Can I _unresolve_ the conversation? I clicked "resolve" while you were commenting, apparently :grimacing:

Reply:
My problem with `MostRecentUpgrade` would be that the `initial_state_root_hash` doesn't have to be related to the upgrade - it is a hash from the block that was the highest one on startup. So what the fields have in common is that they are all related to the current run, but not necessarily to the upgrade...

Reply:
I guess you're right. :+1:
(Sorry, only saw your comments just now.)